actions/src/sema4ai/actions/_table.py

```diff
@@ -15,12 +15,22 @@ class Table(pydantic.BaseModel):
 
     columns: list[str]
     rows: list[Row]
-
-    def __init__(self, columns: list[str], rows: list[Row]):
+    name: str | None = None
```

Comment:
We should not break backward compatibility here, so, this means that if the user doesn't pass a `name` or a `description`, the `model_dump` must not have a `name` nor `description` if they're None. Can you create a test for that?